docs/reference/settings/security-settings.asciidoc

```diff
@@ -78,7 +78,7 @@ The roles to associate with the anonymous user. Required.
 When `true`, an HTTP 403 response is returned if the anonymous user
 does not have the appropriate permissions for the requested action. The
 user is not prompted to provide credentials to access the requested
-resource. When set to `false`, a HTTP 401 is returned and the user
+resource. When set to `false`, HTTP 401 is returned and the user
```

Comment:
I think this reads a little better as "an HTTP 401 is returned"

Reply:
I'll take the wording from `docs/java-rest/low-level/configuration.asciidoc`: "an HTTP 401 **response** is returned".